# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/websockets/extended-payload-length.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/websockets/extended-payload-length.html.ini
--- a/testing/web-platform/meta/websockets/extended-payload-length.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/websockets/extended-payload-length.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,11 +1,13 @@
 [extended-payload-length.html?wpt_flags=h2]
   expected:
-    if (os == "android") and not debug and webrender: ["ERROR", "TIMEOUT"]
-    if (os == "android") and not debug and not webrender: ["ERROR", "TIMEOUT", "CRASH"]
-    if (os == "android") and debug and not webrender: ["TIMEOUT", "ERROR"]
-    if (os == "android") and debug and webrender: ["ERROR", "TIMEOUT"]
+    if (os == "android") and not debug and not webrender: ["TIMEOUT", "ERROR", "CRASH"]
+    if (os == "android") and not debug and webrender: ["ERROR", "TIMEOUT", "CRASH"]
+    if (os == "win") and (processor == "x86") and debug: OK
     if (os == "win") and (processor == "x86") and not debug: ["TIMEOUT", "OK"]
-    if os == "linux": TIMEOUT
+    if (os == "android") and debug: ["TIMEOUT", "ERROR"]
+    if (os == "win") and (processor == "x86_64"): OK
+    if os == "mac": OK
+    TIMEOUT
 
 [extended-payload-length.html?wss]
 